freqtrade/persistence/models.py

```diff
@@ -259,7 +262,29 @@ class LocalTrade():
     strategy: str = ''
     timeframe: Optional[int] = None
 
+    # Margin trading properties
+    leverage: Optional[float] = 0.0
```

Comment:
shouldn't the default for leverage be 1? (non-leveraged?) 

i guess it'll depend on the calculation applied ... just asking ... :laughing: